features/delete_team.feature

```diff
@@ -17,6 +17,9 @@ Scenario: Team leader deletes the team
   Then I should be on the Team Database page
   And I should see "Team kiwi successfully deleted"
   And I should not be on a team
+  When I go to the Database page
```

Comment:
Why do you need to navigate to the Database page first? Can't you just click on Teams in the nav-bar?